plugin/src/test/java/org/opensearch/ml/action/deploy/TransportDeployModelActionTests.java

```diff
@@ -490,10 +500,10 @@ public void testDoExecute_whenThreadPoolExecutorException_TaskRemoved() {
         MLModel mlModel = mock(MLModel.class);
         when(mlModel.getAlgorithm()).thenReturn(FunctionName.ANOMALY_LOCALIZATION);
         doAnswer(invocation -> {
-            ActionListener<MLModel> listener = invocation.getArgument(3);
+            ActionListener<MLModel> listener = invocation.getArgument(4);
             listener.onResponse(mlModel);
             return null;
-        }).when(mlModelManager).getModel(anyString(), isNull(), any(String[].class), Mockito.isA(ActionListener.class));
+        }).when(mlModelManager).getModel(anyString(), any(), isNull(), any(String[].class), Mockito.isA(ActionListener.class));
```

Comment:
picked a random one of these to comment on.   I've been using `nullable(String.class)` for the tenant ID, but whatever you do, be consistent.  `any()` should work unless you get an argument conflict.